client/set-packages.sh

```diff
@@ -1,10 +1,11 @@
 PROJECTS_DIR=${PROJECT_DIR_OVERRIDE:-"../ProjectsSource/"}
-HOME_FOR_BUILD=${HOME_OVERRIDE:-$HOME}
+HOME_FOR_BUILD=${1:-$HOME}
```

Comment:
I think this may break some build scripts on GHA. The buildPr workflow needs to override these, but I guess it could be modified to pass ${{github.workspace}} in as a parameter? 
```yaml
      - name: Install local packages
        working-directory: ./client
        shell: bash
        run: |
          ./set-packages.sh
        env:
          HOME_OVERRIDE: ${{ github.workspace }}
          PROJECT_DIR_OVERRIDE: ${{ github.workspace }}/NugetBuildDir/
```

Reply:
I've attempted this fix; we'll see. If it works I'll change my review to approve.